### PR TITLE
[zh-cn]Fix some layout of force-delete-stateful-set-pod.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Some minor fixes and optimizations
```yaml
content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md
```

### CODE REVIEW

File: content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md [hunk 1]
```diff
@@ -68,15 +68,17 @@ kubectl delete pods <pod>
...
+[shuts down gracefully](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
+before the kubelet deletes the name from the apiserver.
 -->
-为了让上面操作能够体面地终止 Pod，Pod **一定不能** 设置 `pod.Spec.TerminationGracePeriodSeconds` 为 0。
-将 `pod.Spec.TerminationGracePeriodSeconds` 设置为 0s 的做法是不安全的，强烈建议 StatefulSet 类型的
+为了让上面操作能够体面地终止 Pod，Pod **一定不能**设置 `pod.Spec.TerminationGracePeriodSeconds` 为 0。
+将 `pod.Spec.TerminationGracePeriodSeconds` 设置为 0 秒的做法是不安全的，强烈建议 StatefulSet 类型的
 Pod 不要使用。体面删除是安全的，并且会在 kubelet 从 API 服务器中删除资源名称之前确保
-[体面地结束 pod ](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)。
+[体面地结束 Pod ](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md:78]:
```suggestion
[体面地结束 Pod](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)。
```

File: content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md [hunk 2]
```diff
@@ -86,14 +88,14 @@ The only ways in which a Pod in such a state can be removed from the apiserver a
 当某个节点不可达时，不会引发自动删除 Pod。
 在无法访问的节点上运行的 Pod 在
 [超时](/zh-cn/docs/concepts/architecture/nodes/#condition)
-后会进入'Terminating' 或者 'Unknown' 状态。
+后会进入 'Terminating' 或者 'Unknown' 状态。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md:91]:
```suggestion
后会进入 “Terminating” 或者 “Unknown” 状态。
```

File: content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md [hunk 3]
```diff
@@ -103,10 +105,11 @@ The only ways in which a Pod in such a state can be removed from the apiserver a
 <!--
 The recommended best practice is to use the first or second approach. If a Node is confirmed to be dead (e.g. permanently disconnected from the network, powered down, etc), then delete the Node object. If the Node is suffering from a network partition, then try to resolve this or wait for it to resolve. When the partition heals, the kubelet will complete the deletion of the Pod and free up its name in the apiserver.
 -->
-推荐使用第一种或者第二种方法。如果确认节点已经不可用了 (比如，永久断开网络、断电等)，
+推荐使用第一种或者第二种方法。
+如果确认节点已经不可用了（比如，永久断开网络、断电等），
 则应删除 Node 对象。
-如果节点遇到网裂问题，请尝试解决该问题或者等待其解决。
-当网裂愈合时，kubelet 将完成 Pod 的删除并从 API 服务器上释放其名字。
+如果节点遇到网络分区问题，请尝试解决该问题或者等待其解决。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md:111]: The original is better.

File: content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md [hunk 4]
```diff
@@ -103,10 +105,11 @@ The only ways in which a Pod in such a state can be removed from the apiserver a
...
+当网络分区愈合时，kubelet 将完成 Pod 的删除并从 API 服务器上释放其名字。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md:112]: The original is better.

**Pull request merged**